coordinator/src/main/scala/com/socrata/datacoordinator/common/util/DatasetIdNormalizer.scala

```diff
@@ -0,0 +1,26 @@
+package com.socrata.datacoordinator.common.util
+
+import com.ibm.icu.text.Normalizer
+import com.rojoma.json.v3.io.{IdentifierEvent, FieldEvent, StringEvent, JsonEvent}
+
+object DatasetIdNormalizer {
+
+  val normalizationMode: Normalizer.Mode = Normalizer.NFC
```

Comment:
Do we ever plan to use any other Normalizer other then NFC?  Should we make this configurable?  Should we have a class Hierarchy where there is a ```BaseNormalizer``` and a subclass of ```NFCNormalizer```?